packages/hash/api/src/model/knowledge/page.model.ts

```diff
@@ -29,6 +28,20 @@ type PageModelCreateParams = Omit<
  * @class {@link PageModel}
  */
 export default class extends EntityModel {
+  constructor(pageEntityModel: EntityModel) {
```

Comment:
It's strange to me that this takes `EntityModel`, maybe we should delete `fromEntityModel`, or change this to take the destructured contents

Reply:
Removed in 1e71a8382db6021f203e23349af1b883ed8f3540